ironic.conf

```diff
@@ -5,12 +5,14 @@ default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
+default_raid_interface = no-raid
 enabled_boot_interfaces = pxe,ipxe,fake
 enabled_deploy_interfaces = direct,fake
 enabled_hardware_types = ipmi,idrac,irmc,fake-hardware,redfish
 enabled_inspect_interfaces = inspector,idrac,irmc,fake,redfish
 enabled_management_interfaces = ipmitool,idrac,irmc,fake,redfish
 enabled_power_interfaces = ipmitool,idrac,irmc,fake,redfish
+enabled_raid_interfaces = no-raid,irmc,agent
```

Comment:
We seem to add `fake` everywhere (presumably for testing). Let's add it here as well.

Reply:
Done :)
Thank you very much :)